### ISSUE TITLE
Design: Update Style Guide Presentation with newer design changes

### ISSUE THREAD

**Issue by NilakshiS** [hackforla/tdm-calculator#1998]
### Overview
We need to update the slide deck with new content, because the style guide presentation is missing some of our newer designs.

### Details
Refer to this List of issues and notes on changes that will need to get added to the style guide
- Icons:
  - Admin Notes, note icon #1889 
  - Sorting icons, up and down #1919 
- New Modals from #1743 
- Drop Shadow styles made by Grace & Nami from #1743 
- Possible updates to warning infobox based on issue #1935 
- Color changes
  -  Warning color removed? `#C35302`
  - Add new dropbox, checkboxes, and radio button colors from #1839 

### Action Items

- [x] Update these existing slides:
  - [x] Add to icons - Admin Notes, note icon
  - [x] Add to icons - Sorting icons, up and down
  - [x] Update Modals - New modal designs
  - [x] Update color with new dropbox,checkbox, radio button colors
- [x] Create new slides for these:
  - [x]  Drop Shadows
- [x] Get approval on changes from design
- [x] Review with product and dev. Update if needed based on feedback
- [x] Once finalized, add before and after images to the [staging deck on this slide]
   - [x] if no slide has been made, make one in [the staging deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
- [ ] Get Stakeholder sign-off via the stakeholder meeting slide deck.

### Resources/Instructions
- [Link to the style guide presentation](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit)
- [Design System on Figma](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=3337-11981&t=qPL08THnOxegPiv9-1)

#### Slides
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
   - ~[Staging Deck, slide 1998](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?pli=1#slide=id.g328276ff956_0_0)~
      - [staging deck, issue 1998, slide 1](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g35c7465f46c_2_24#slide=id.g35c7465f46c_2_24)
      - [staging deck, issue 1998, slide 2](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g35d66aff36a_4_15#slide=id.g35d66aff36a_4_15)
      - [staging deck, issue 1998, slide 3](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g35d66aff36a_6_0#slide=id.g35d66aff36a_6_0)
      - [staging deck, issue 1998, slide 4](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g35d66aff36a_4_29#slide=id.g35d66aff36a_4_29)
      - [staging deck, issue 1998, slide 5](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g361502f6416_1_0#slide=id.g361502f6416_1_0)
      - [staging deck, issue 1998, slide 6](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g35eb1de5978_0_0#slide=id.g35eb1de5978_0_0)
      - [staging deck, issue 1998, slide 7](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g35c7465f46c_2_50#slide=id.g35c7465f46c_2_50)
      - [staging deck, issue 1998, slide 8](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g361502f6416_1_9#slide=id.g361502f6416_1_9)
      - [staging deck, issue 1998, slide 9](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g361502f6416_1_35#slide=id.g361502f6416_1_35)
- Release Deck
   - [YYYY-MM-DD Release Deck, issue 1998, slide 1]
   - [YYYY-MM-DD Release Deck, issue 1998, slide 2]
   - [YYYY-MM-DD Release Deck, issue 1998, slide 3]
   - [YYYY-MM-DD Release Deck, issue 1998, slide 4]
   - [YYYY-MM-DD Release Deck, issue 1998, slide 5]
   - [YYYY-MM-DD Release Deck, issue 1998, slide 6]
   - [YYYY-MM-DD Release Deck, issue 1998, slide 7]
   - [YYYY-MM-DD Release Deck, issue 1998, slide 8]
   - [YYYY-MM-DD Release Deck, issue 1998, slide 9]

**Comment by hnoriega27** [comment]
Test

**Comment by thetanmancan** [comment]
Hi @hnoriega27. When you get a chance, could you please provide a progress update for this issue? Thank you.

**Comment by hnoriega27** [comment]
![Image](https://github.com/user-attachments/assets/6bc25639-6dbb-47de-befd-2f616eb8387f)

![Image](https://github.com/user-attachments/assets/997d8460-eeda-4958-8b57-d5b42ddd9926)

@thetanmancan Thanks for reaching out! I’ve completed the slides above so far. I’ll continue updating the module section and add a new section to the style guide for shadows later today (3/5) before our meeting.

**Comment by marlenamellody** [comment]
Feedback from 2025-03-05 design meeting: 

- [ ] Remove duplicate icons (notes, filter)
- [ ] Remove icons that are not in the system (other filter icons)
- [ ] Make sure links navigate to specific icon
- [ ] Add a note to the sorting icons indicating that the icon is rotated 
- [ ] Update modals slide with latest designs from [Figma](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=16595-1406&t=tqNZTDblAz8Rftfh-4)

**Comment by thetanmancan** [comment]
Hi @hnoriega27. When you get a chance, could you please provide an update on this issue using the format below? Thank you.

1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."

* remember to add links to the top of the issue if they are going to be needed again.

**Comment by senakim18** [comment]
Hi @thetanmancan , do we have an updated ETA for this?

**Comment by thetanmancan** [comment]
Hi @senakim18. I have not heard from Hansel.

@hnoriega27, could you please provide an update and ETA for this issue? Thank you.

**Comment by marlenamellody** [comment]
Hi @hnoriega27, thanks for making those changes. Here are some slides that still need to be updated: 

- [Color palette slide (7)](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g2e134a56948_3_13#slide=id.g2e134a56948_3_13): remove the warning color `#C35302`, add the radio button/dropdown blue `#1967D2` and `#1450A4`, and update error background color `#FFEDEA`
- [Drop shadow slide (26)](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g341b99c9aa5_0_40#slide=id.g341b99c9aa5_0_40): refer to [Figma regarding drop shadow styles](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=16316-134373&t=Ld8fRAoEcIhyJItA-4) and update the screenshot on the slide to the updated modal style 
- [Icons slide 17](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g33a63bbf9e0_0_0#slide=id.g33a63bbf9e0_0_0) make sure links navigate to the specific icon (example: [chevron right](https://mui.com/material-ui/material-icons/?query=s))

Let me know if you have any questions!

**Comment by hnoriega27** [comment]
@ExperimentsInHonesty 

1) Progress: Provided another update to #issue 1998
2) Blockers: None
3) Availability: 2-3 hours during the week. I am available to meet for additional meetings but would need to have advance notice to adjust my work schedule
4) ETA: Updated provided
5) Pictures (if necessary): Attached to the issue

**Comment by hnoriega27** [comment]
![Image](https://github.com/user-attachments/assets/7a4b744b-5da7-4a88-9cf2-54b1b6b56aac)

Updated the Color Palette slide

**Comment by hnoriega27** [comment]
![Image](https://github.com/user-attachments/assets/95b86c0c-3dc5-41ee-aa26-879438e2ee31)

Slide (27): Updated the Drop Shadow Styles slide. I made a note to distinguish each design element for designers

**Comment by hnoriega27** [comment]
![Image](https://github.com/user-attachments/assets/b6943f5d-d6e4-4770-a850-64c94f299fba)

Slide (17): Updated the Iconography slide. I was not able to find two icons (notes and expand) to reference on the materials url. So I left the url as is. We can discuss it in more detail in our team meeting.

**Comment by hnoriega27** [comment]
@marlenamellody Thanks for the detailed notes! I uploaded the revision. Please let me know if there are any additional changes.

**Comment by thetanmancan** [comment]
Hi @marlenamellody and @hnoriega27. I took a brief look at the Style Guide Presentation on Wednesday. Due to limited time, I only looked at the first eight slides, and I'm not sure if every slide on the deck is within the scope of this issue. If not, please let me know which specific slides I should be looking at.

Anyhow, here are the notes I made. Marlena, let me know if you agree with these or not.

- Slide 3 | Table of Contents
  - Needs to be updated with accurate slide numbers.
- Slide 7 | Color Palette
  - How are we defining Primary, Secondary, Grayscale, and Semantic colors? I’m not sure if the Radio and Dropdown colors belong in Semantic. Depends on how they’re defined.
  - Are we still using Dark Blue in our system? If we are, I don’t know where it’s being used. If we aren’t, it should be removed.
  - The descriptions “Design elements” and “Design element for menu selection” are not specific like the rest of the color descriptors. Please make them more specific to how they are being used.
- Slide 8 | Color Palette
  - Do we need these examples? I don’t think they add much value. Unless we’re going to be exhaustive about which colors apply to which elements in our site, I’m not sure what the point of this slide is.
  - If we do decide to keep this slide:
    - The main text reads like it’s instructive and meant for designers still working on the project. This presentation is meant to be informative for stakeholders and whoever ends up working on the TDM website after we hand it off to LADOT. Please change the style to explain what the standards are that we have, not what work needs to be done in our style guide or website.
    - The colors described are no longer accurate and should be updated.

**Comment by thetanmancan** [comment]
Adding [this comment](https://github.com/hackforla/tdm-calculator/issues/1885#issuecomment-2754499995) from last week that was supposed to be posted in this issue's comments but were mistakenly posted in #1885.

**Comment by marlenamellody** [comment]
Hi @thetanmancan and @hnoriega27, this issue focuses on updating [icons](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g33a63bbf9e0_0_0#slide=id.g33a63bbf9e0_0_0), the [color palette](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g2e134a56948_3_13#slide=id.g2e134a56948_3_13), [modals](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g332dfe44484_0_6#slide=id.g332dfe44484_0_6), [infoboxes](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g2f74a24ec8f_0_8#slide=id.g2f74a24ec8f_0_8), and[ drop shadows](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g347b0193470_3_0#slide=id.g347b0193470_3_0), as they haven't been updated to reflect our newer designs.  

- I agree that slide 3 should be updated with accurate slide numbers

> Slide 7 | Color Palette
How are we defining Primary, Secondary, Grayscale, and Semantic colors? I’m not sure if the Radio and Dropdown colors belong in Semantic. Depends on how they’re defined.
Are we still using Dark Blue in our system? If we are, I don’t know where it’s being used. If we aren’t, it should be removed.
The descriptions “Design elements” and “Design element for menu selection” are not specific like the rest of the color descriptors. Please make them more specific to how they are being used.

- Good question. Semantic colors usually communicate status (like success, error, or warning). If the colors are tied to status or user feedback, they could fall under semantic—but I’m not sure that applies in this case, since the radio and dropdown components use blue rather than green or red. @hanselnoriega please let us know what you think.
- I don't recall Dark Blue being used anywhere in our system so it should be removed.
- Hansel updated the descriptors on the slides. @hanselnoriega, it seems like the color swatches are switched. `#1967D2` should be the lighter blue for default state, and `#1450A4` for hover state. 

> Slide 8 | Color Palette
Do we need these examples? I don’t think they add much value. Unless we’re going to be exhaustive about which colors apply to which elements in our site, I’m not sure what the point of this slide is.
If we do decide to keep this slide:
The main text reads like it’s instructive and meant for designers still working on the project. This presentation is meant to be informative for stakeholders and whoever ends up working on the TDM website after we hand it off to LADOT. Please change the style to explain what the standards are that we have, not what work needs to be done in our style guide or website.
The colors described are no longer accurate and should be updated.

- I think this slide would be useful for explaining the standards we have. Maybe it could also include how we define primary, secondary, grayscale, and semantic colors?

**Comment by ExperimentsInHonesty** [comment]
@hanselnoriega Please provide update

<details><summary>Instructions</summary>

1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."
</details> 

1. Progress: 
2. Blockers: 
3. Availability:
4. ETA:
5. Pictures (if necessary):

**Comment by hnoriega27** [comment]
@ExperimentsInHonesty 

1. Progress: Provided another update to #issue 1998
2. Blockers: None
3. Availability: 2-3 hours during the week. I am available to meet for additional meetings but would need to have advance notice to adjust my work schedule
4. ETA: Updated provided
5. Pictures (if necessary): Attached to the issue

**Comment by thetanmancan** [comment]
Hi @hnoriega27. Please post your updates on this issue rather than in #1885, as you did with [this comment](https://github.com/hackforla/tdm-calculator/issues/1885#issuecomment-2791181946), [this comment](https://github.com/hackforla/tdm-calculator/issues/1885#issuecomment-2791182833), and [this comment](https://github.com/hackforla/tdm-calculator/issues/1885#issuecomment-2791183363).

**Comment by thetanmancan** [comment]
Hi @hnoriega27 and @marlenamellody. 

Is this ready for me to look at? If not, is there an ETA for when I should take a look at this?

Judging by [this comment](https://github.com/hackforla/tdm-calculator/issues/1998#issuecomment-2784809528), I assume this issue isn't meant to cover every slide on the presentation. If this issue is ready for review, can you please provide a list of all the slides I should be taking a look at, Hansel?

**Comment by hnoriega27** [comment]
![Image](https://github.com/user-attachments/assets/6104e1d5-3dca-4c2c-83bb-3860dce0f68e)

@thetanmancan // cc @marlenamellody 
The following are slides that are ready to be reviewed: Slide 03, Slide 07, and Slide 08. 
Pending work still in progress are for issue #1885. As of right now (4/10/25) I'm about halfway done with the changes  for this issue but will post here for the final updates for review. The ETA, I have to completed issue #1885 is by tomorrow 12 pm PST. Currently, I am traveling for work out of the country but making this a priority.

Slide 03 | Table of Context:
Updated the table of contexts to match the new slides.

**Comment by hnoriega27** [comment]
![Image](https://github.com/user-attachments/assets/a046bb8a-0e49-4fa6-a844-2263d8315ccb)

Slide 07 | Color Palette:
Updated the color box to properly reflect #1967D2 and #1450A4. Good catch! I agree, the drop-down menu and radio buttons are fine as navy blue. No need to change it to another color, especially as it falls under secondary color to highlight contrast. Unless it's grayscale? Either way, I think it's fine as navy blue.

**Comment by hnoriega27** [comment]
![Image](https://github.com/user-attachments/assets/7ea9c528-1121-4a2f-9040-113d5c5b4aef)
![Image](https://github.com/user-attachments/assets/9d0fef24-33f2-476c-810c-62a066828910)

Slide 08 | Color Palette:
Agree—expanding on the definitions of primary, secondary, grayscale, and semantic colors is valuable, as it helps establish clarity on the proper terms and usage. Definitions can help provide a better understanding of the design system to other people (stakeholders, teams) besides designers. I took some time to create the following slides. These slides are not published but would like confirmation and feedback in order to move forward.

**Comment by marlenamellody** [comment]
> ![Image](https://github.com/user-attachments/assets/a046bb8a-0e49-4fa6-a844-2263d8315ccb)
> 
> Slide 07 | Color Palette: Updated the color box to properly reflect #1967D2 and #1450A4. Good catch! I agree, the drop-down menu and radio buttons are fine as navy blue. No need to change it to another color, especially as it falls under secondary color to highlight contrast. Unless it's grayscale? Either way, I think it's fine as navy blue.

- I don't recall  'Dark Blue' `#00122B` under secondary colors being used anywhere our system so it should be removed unless you find an instance where it's still in use.
- The selection controls: checkboxes, radio buttons, and dropdowns (hover, default) are different blues. When you mention 'Navy blue', is that what you're referring to? And do these colors still fall under semantic colors?

**Comment by hnoriega27** [comment]
![Image](https://github.com/user-attachments/assets/6e0c0191-c96f-4de0-8500-618c3a46154e)

Post for changes on #1885 will be addressed here. @thetanmancan // cc @marlenamellody 

1) Update the table headers
2) Update the filter icons
3) Update the sort buttons
4) Update the primary button style and text
5) Update the date picker UI
6) Update the pagination UI
7) Update the connections and components

----
1) Update the table headers | Slide 52
Updated the slide with new images

**Comment by hnoriega27** [comment]
![Image](https://github.com/user-attachments/assets/8ac6adc6-9e76-4fd3-8ab3-9cddc70f636f)

2) Update the filter icons | Slide 15 
Slide updated with filter icons, condensed and organized by Marlena. Thank you.

3) Update the sort buttons | Slide 15
Slide updated with filter icons, condensed and organized by Marlena. Thank you.

**Comment by hnoriega27** [comment]
4) Update the primary button style and text | Slide ?
I had a hard time confirming this information from the Figma file that was linked to issue 1885. I was wondering if the information has been updated already or can you confirm?

**Comment by hnoriega27** [comment]
![Image](https://github.com/user-attachments/assets/90d5b32e-b52f-4cdf-9de5-16caaec57ea0)

5) Update the date picker UI | Slide 48
I updated the changes on this slide.

**Comment by hnoriega27** [comment]
![Image](https://github.com/user-attachments/assets/5b4a5d07-6a23-483b-bed5-ade6c7dbcf35)

6) Update the pagination UI | Slide 12
It looks like the pagination has been updated but confirming this is correct.

**Comment by hnoriega27** [comment]
7) Update the connections and components | Slide ?
I am not sure about this update. Would you please advise me or give me more direction.

**Comment by hnoriega27** [comment]
@marlenamellody > * I don't recall  'Dark Blue' `#00122B` under secondary colors being used anywhere our system so it should be removed unless you find an instance where it's still in use.
> * The selection controls: checkboxes, radio buttons, and dropdowns (hover, default) are different blues. When you mention 'Navy blue', is that what you're referring to? And do these colors still fall under semantic colors?

I updated the context and page numbers. As for the 'Dark Blue' `#00122B`, I removed that color. There was a mentioned of it from "navy blue" from a conversation that Tristian asked about radio buttons and drop down menu but I believe it does fall under semantic colors. You are right about that.

**Comment by ExperimentsInHonesty** [comment]
@hnoriega27 What is the status of this issue.  I see a comment from you above, but its unclear if @marlenamellody saw it because there was no `ready for design lead - Marlena` label added and it was not moved to the Questions column.  If it does require her attention, please take those actions.  If it does not, please write next steps and ETA for you finishing this issue.

**Comment by marlenamellody** [comment]
@hanselnoriega, 

> Update the primary button style and text | Slide ?
I had a hard time confirming this information from the Figma file that was linked to issue 1885. I was wondering if the information has been updated already or can you confirm?

This has already been updated.

> Update the connections and components | Slide ?
I am not sure about this update. Would you please advise me or give me more direction.

This is related to the My Projects prototype, so there's no slide to update.

**Comment by hnoriega27** [comment]
![Image](https://github.com/user-attachments/assets/663c774e-1450-43b1-8dff-05339bf25a60)

@thetanmancan // cc @marlenamellody 

Updated the slide for review!

**Comment by thetanmancan** [comment]
@marlenamellody, I believe the Style Guide Presentation needs more updates before this issue can be closed. Please take a look at my comments below and, if you agree with them, please let @hnoriega27 know.

- Slide 7
  - The Dark Navy and Link Blue colors are not vertically aligned with the border of the description box next to them.
  - The Green description is missing its gray border.
  - The name of the new blue colors, “Selection Control Default State” and “Selection Control Hover State” are very long, causing the description box to be tall enough to be inconsistent with every other color on the page. Can we change the names to something shorter, like “Selector Blue” and “Hover Blue”, or something along those lines?
- Guidelines for UI Color Palette
  - I think we can leave out the part about emotional engagement since we’re not trying to convert customers and this is a pretty dry, functional web app.
  - This says “Primary Colors”, but we only have one primary color.
  - This claims that the usage of primary colors is high-priority elements such as buttons, links, and highlights, but our Navy Blue is used for none of those things.
  - Please go through and rewrite the definitions and usage of each of the color types so that they make sense with what is shown on Slide 7. Feel free to rearrange or rename the color categories on Slide 7 in a way that makes more sense.
- Slide 12
  - What is a sibling count? Please explain on the slide and show examples.
  - I don’t recall anywhere in our system where we use the Secondary Left or Secondary Right buttons. If there is somewhere, please let me know. If there isn’t, can we please remove those buttons from our design system?
  - I don’t know why there is a “Primary” category for the Text button when there’s no Secondary. Can we just call them “Text Button” and “Text Button with Chevron”?
  - The icon for Expand All looks different in our system than it does on this slide. This slide should be updated with the correct icon.
  - Please rewrite the text in the bottom-right corner so that it’s clearer to understand. It’s a bit clunky right now.
- Slide 15
  - FitlerAlt’s “Active” is missing an end parenthesis.
  - There’s no space after “Switch Right - ascending”.
  - For the sort icons, please specify which direction they are rotated 90 degrees.
  - I believe the “Expand” icon is different from browser to browser, so I don’t think this should be part of our design system.
  - The names for these icons need to be updated. For example, I checked Material Design and what we currently have as “FilterAlt” is shown as “filter_alt” in Material Design’s Figma Community file and “Filter Alt” on their Google Fonts site. We should pick a naming convention and then be consistent with it. Since @JaimeCucalon is working on updating our design system as part of #2227, this is probably a conversation worth having with him.
  - Can you please consult with @JaimeCucalon on #2227 and do some kind of audit to make sure we’re still using all these icons on our site? I’m guessing some of them are outdated, but I wouldn’t know without diving in and checking myself.
- Slides 47, 48
  - This is all very confusingly organized at the moment, both on Figma and these slides.
  - I would say the first slide should show examples of how each control would look on a page.
  - The second slide would show how colors are applied to them according to their state. I recommend showing a color matrix showing the difference between Unselected and Selected states on one axis and Default and Hover states on a different axis.
  - Please update the Date UI Picker with all possible combinations of text weight, text color, and background color.
- Slide 52
  - The Active (Selected) image does not show page selection. Please update that.
  - The footer does not show the Glossary or Checklist links. Please update that.

**Comment by marlenamellody** [comment]
@thetanmancan thanks for reviewing this issue–I agree. @hanselnoriega, please make the necessary changes. I know there's a lot to update so I recommend going through the dev site and reviewing the design system to understand the changes that need to be made. As always, reach out if you have any questions or concerns.

**Comment by marlenamellody** [comment]
@hnoriega27, please provide an update: 

Progress:
Blockers:
Availability:
ETA:
Pictures (if necessary):

**Comment by hnoriega27** [comment]
@marlenamellody // cc @thetanmancan Just arrived last night 4/21/25 from a 16-hour flight. Here are the updates below.

Progress: Currently uploaded
Blockers: Slide 15 to collab with JaimeCucalon for a cohesive naming convention. Slide has been recently updated but pending for questions.
Availability: 3-4 per week
ETA: Updated on 4/22
Pictures (if necessary): Attached

**Comment by hnoriega27** [comment]
![Image](https://github.com/user-attachments/assets/b5dfd7ed-508c-48be-9905-0c7e1030cdc2)
![Image](https://github.com/user-attachments/assets/8e6e7370-aa85-4842-9300-32fe9715182b)

**Slide 9 + 10**
Removed - emotional engagement
Updated - "Primary Colors" text
Rephrased - This claims that the usage of primary colors is high-priority elements such as buttons, links, and highlights, but our Navy Blue is used for none of those things.
Updated the rest of the terminology

**Comment by hnoriega27** [comment]
![Image](https://github.com/user-attachments/assets/3dc9220b-a39a-47ff-9196-a944bc5cd80f)

4/22 - Slide 14 (Formerly Slide 12)
I'm not sure what the sibling count is, either. I made an attempt to look over our Figma design prototypes and design systems, but was not able to find information to reference this section. 

Removed - Secondary Left or Secondary Right buttons, I was able to find the buttons mentioned but this design was from [last year Figma Issue](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=18525-35&p=f&t=ndhdOIf7Xw7s2IVc-0). Currently, from what I am able to tell, I don't see these UI elements used, unless I am mistaken the gray arrow page buttons were used last year. 

Updated - “Primary” category for the Text button when there’s no Secondary. Can we just call them “Text Button” and “Text Button with Chevron”? 

Updated - The icon for Expand All looks different in our system than it does on this slide. This slide should be updated with the correct icon. 

Updated - Please rewrite the text in the bottom-right corner so that it’s clearer to understand. It’s a bit clunky right now.

**Comment by hnoriega27** [comment]
![Image](https://github.com/user-attachments/assets/79b5a755-69d9-4b45-9c00-0cce4d83b638)

**Slide 17 (Formerly Slide 15)**

Resolved the following.

FitlerAlt’s “Active”. Resolved 
No space after “Switch Right - ascending”. 
Sort icons, please specify which direction they are rotated 90 degrees.

Removed, the “Expand” icon. Also, just messaged JameCucalon to collab on updating the design system and a cohesive naming convention

**Comment by hnoriega27** [comment]
![Image](https://github.com/user-attachments/assets/23fb6515-2c63-402c-84ad-d768715b5180)
![Image](https://github.com/user-attachments/assets/13adbb53-3fb6-4e5f-91f9-c75e3ce15c90)

Slide 49 + 50 (Formerly Slide 47 + 48)
Updated - I typically don't like to reference design work that is currently pending but these are the limited examples that I was able to find for the bullet points to correct [Figma File](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=18525-32&p=f&t=ndhdOIf7Xw7s2IVc-0)

**Comment by ExperimentsInHonesty** [comment]
@hanselnoriega this screenshot is too blurry. Please provide a new one, and if unable, please provide the figma link to the exact location in the figma file where we can make one.

**Comment by marlenamellody** [comment]
Hi @hanselnoriega, thanks for your updates. 

- Is [this slide](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g2a6d1bfeb46_0_166#slide=id.g2a6d1bfeb46_0_166) still necessary since we have slides for color guidelines? If so, the information needs to be updated. 
   -  white and black don't fall under primary colors in our system
   -  yellow is not used for AIN/PIN
   - grays are not secondary 
- [Slide 14](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g2e134a56948_6_41#slide=id.g2e134a56948_6_41) 
  - "Text button with chevron" icon should match the icon in the examples. 
- [Slide 17](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g2e77662f2ce_0_10#slide=id.g2e77662f2ce_0_10)
   - For the sort icons, please specify which direction they are rotated 90 degrees.
- [Slide 49](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g305f865959f_0_11#slide=id.g305f865959f_0_11) I'd recommend going through the filter overlays on the [my projects page](https://tdm-dev.azurewebsites.net/projects) and refering to the [Figma](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=16061-4932&t=qZTGX4dlXNjf391r-4)  
  - The Unselected state is different from the Default state. 
  - The Hover state color should be `#1450A4`. The example screenshots are using the Default state `#1967D2` for checkboxes and dropdowns. 
 <details><summary>Unselected State Example</summary> 

<img width="239" alt="Image" src="https://github.com/user-attachments/assets/1e81b351-2ed2-490e-8305-d3ba0b34feda" />
</details>
 <details><summary>Default State Example</summary> 
<img width="238" alt="Image" src="https://github.com/user-attachments/assets/5bda3df4-5d99-4a19-8913-28242b469eeb" /></details>

 <details><summary>Hover State Example</summary> 

<img width="200" alt="Image" src="https://github.com/user-attachments/assets/5768091f-e0b6-466c-9563-b09458c413aa" />

</details>

**Comment by thetanmancan** [comment]
Hi @marlenamellody. Since Hansel has been offboarded, I'm moving this issue back to the Prioritized Backlog.

**Comment by georgeivasquez** [comment]
Hi team, I've been able to make updates according to the instructions. This is my first issue, so please let me know if I have missed anything in my updates. 

https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=18525-42&p=f&t=cBMxI9H0O8B3XPfp-0

**Comment by lola3736** [comment]
@georgeivasquez is this ready for peer review? can get feedback from team members to help ensure you are on the right track.

**Comment by georgeivasquez** [comment]
Commented on the Slack but yes

**Comment by ExperimentsInHonesty** [comment]
@georgeivasquez will put before and after images into the issue comments, so that I can review.  When done, add the ready for product label and move to visual review.

**Comment by georgeivasquez** [comment]
Hi @ExperimentsInHonesty,
I've uploaded the Before and After versions for each issue in the style guide, organized according to the sequence of instructions.

Let me know if you’d like me to make any changes. Thanks!

[Issue #1889 Admin Notes, Note Icon:](https://github.com/hackforla/tdm-calculator/issues/1889)

Before:

<img width="246" alt="Image" src="https://github.com/user-attachments/assets/bb311195-2ec2-495f-ae74-59edd5673f0e" />

After:

Admin Notes

<img width="377" alt="Image" src="https://github.com/user-attachments/assets/8a8d8e84-d440-445b-a0de-f1387e16c9dd" />

Note Icon

<img width="124" alt="Image" src="https://github.com/user-attachments/assets/4cacc620-1d89-439b-ab7e-99b18f3cc933" />

[Issue #1919 Sorting Icons ](https://github.com/hackforla/tdm-calculator/issues/1919)

Before: 

<img width="325" alt="Image" src="https://github.com/user-attachments/assets/74569326-60d2-46d3-bf2e-d777ade51eb4" />

After:

<img width="671" alt="Image" src="https://github.com/user-attachments/assets/6b4d2648-819c-45b2-a594-090d9ea72aac" />

[Issue #1743 New Modals ](https://github.com/hackforla/tdm-calculator/issues/1743)

Before:

<img width="423" alt="Image" src="https://github.com/user-attachments/assets/1b0b8bc7-6388-4e2c-9194-83bd4d75afdd" />

After:

<img width="325" alt="Image" src="https://github.com/user-attachments/assets/c6a58115-d80d-410e-af32-6948c9507a40" />

Drop Shadow on 1743:

<img width="317" alt="Image" src="https://github.com/user-attachments/assets/00a8896d-80b6-4fca-9a7d-f7a93e2d2013" />

[Issues #1935 Possible updates to warning infobox based on issue](https://github.com/hackforla/tdm-calculator/issues/1935)

<img width="577" alt="Image" src="https://github.com/user-attachments/assets/c3b1d560-383d-462b-baa9-8f59062da287" />

Color Changes 

Remove #C35302

Before: 

<img width="196" alt="Image" src="https://github.com/user-attachments/assets/b28a9322-6b61-4284-9e7d-215c159483e1" />

After:

<img width="343" alt="Image" src="https://github.com/user-attachments/assets/8c72ce65-4334-4eb7-8541-4d51e0cbc15b" />

[Issue 1839 Add new dropbox, checkboxes, and radio button](https://github.com/hackforla/tdm-calculator/issues/1839)

<img width="391" alt="Image" src="https://github.com/user-attachments/assets/0930043e-8e47-4079-ae3a-540887f5518c" />

**Comment by georgeivasquez** [comment]
Screenshots have been updated @ExperimentsInHonesty

**Comment by georgeivasquez** [comment]
@ExperimentsInHonesty Reviewed and Revised!

**Comment by ExperimentsInHonesty** [comment]
@georgeivasquez I am confused by the image below.  I thought there was only one kind of information modal (blue with an i icon).  All other modals are either confirmation (green) or warning lables (soft warning red triangle with green button, and hard warning red triangle and red button).  So where is this new information modal icon coming from?

![Image](https://github.com/user-attachments/assets/befe297e-dd3f-4e29-99e5-b0ef9dc8c55d)

**Comment by georgeivasquez** [comment]
@ExperimentsInHonesty Apologies I put the wrong module on this specific issue after reexamining I found these (listed below) which are said modules listed above only with different spacing can update accordingly 

<img width="593" alt="Image" src="https://github.com/user-attachments/assets/e7c4351f-0056-43ce-8e81-c69fa09836d3" />

**Comment by georgeivasquez** [comment]
@ExperimentsInHonesty Apologies added the wrong module on this specific issue found these (picture below) will update accordingly...

<img width="983" alt="Image" src="https://github.com/user-attachments/assets/ad209dff-4a25-49ce-9c30-421b015a3e38" />

**Comment by georgeivasquez** [comment]
@ExperimentsInHonesty Updated! Should be ready for staging deck

**Comment by ExperimentsInHonesty** [comment]
I am not seeing the changes in the style guide.  This is an example, Jamie will give you the rest of the notes in a new comment.

[Style Guide, Semantic colors, Slide 7](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g2e134a56948_3_13#slide=id.g2e134a56948_3_13)

It should show the after but its showing the before
![Image](https://github.com/user-attachments/assets/4a339031-d51c-4289-b52e-ec7f21440718)

The Figma page for the [Figma design system, colors](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=16061-5221&t=cfkAJEoGeYrobN7u-0) is showing the before as well. 

The only place I am seeing the after is in your Figma page for this issue  [#1998 | Update Style Guide Presentation | George](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=18525-42&p=f&t=cfkAJEoGeYrobN7u-0).

I need you to update both the Figma Design Systems page and the Style Guide

**Comment by georgeivasquez** [comment]
Was able to connect with @jbethrodriguez  and decided the best approach would be to organize the screenshots into three sections: Before and After, and Newly Added Elements.

Below you’ll find all the relevant screenshots:


🔄 ####BEFORE AND AFTER

[Issue #1919 – Sorting Icons](https://github.com/hackforla/tdm-calculator/issues/1919)

Before:

<img width="845" alt="Image" src="https://github.com/user-attachments/assets/db9771c8-06f4-44b8-952a-337010486ef2" />
After:

<img width="1069" alt="Image" src="https://github.com/user-attachments/assets/029e600e-a680-45bf-9bca-635b35f66e99" />

Removed #C35302

Before:

<img width="319" alt="Image" src="https://github.com/user-attachments/assets/6a9d1e97-a48d-4e9e-9e73-889346cf7ffe" />

After:

<img width="257" alt="Image" src="https://github.com/user-attachments/assets/88d73feb-3bce-41a0-bc3f-2e419e8065cc" />


✨ #####NEWLY ADDED ELEMENTS

[Issue #1889 – Admin Notes & Note Icon](https://github.com/hackforla/tdm-calculator/issues/1889)

Admin Notes:

<img width="1123" alt="Image" src="https://github.com/user-attachments/assets/687af860-bf4b-44a8-91e6-586868c75ffc" />

Note Icon:

<img width="124" alt="Image" src="https://github.com/user-attachments/assets/4cacc620-1d89-439b-ab7e-99b18f3cc933" />

New Modals (styled per [Issue #1743](https://github.com/hackforla/tdm-calculator/issues/1743))

<img width="1297" alt="Image" src="https://github.com/user-attachments/assets/6a4fe984-d72d-4047-8878-6880bab093f2" />
Drop Shadow for #1743:

<img width="1264" alt="Image" src="https://github.com/user-attachments/assets/063f5d1a-8dde-4034-8aa6-f303a80c4545" />
[Issue #1935 – Updates to Warning Infobox](https://github.com/hackforla/tdm-calculator/issues/1935)

<img width="758" alt="Image" src="https://github.com/user-attachments/assets/db664a56-868f-402e-a3f9-722bd3730acd" />
[Issue #1839 – New Dropdown, Checkboxes, and Radio Button](https://github.com/hackforla/tdm-calculator/issues/1839)

<img width="1090" alt="Image" src="https://github.com/user-attachments/assets/ad0a32fa-44c7-450a-82a6-2fdf47c26e32" />

📁 Additional Notes

The Design Systems file on Figma still contains the original untouched style guide.

All style guide-related changes for this issue were made on the Issue Layer only.

Will update Design System file once this issue and closed and we move forward with these changes.

Thank you!

**Comment by ExperimentsInHonesty** [comment]
The next steps are
@georgeivasquez 
- [ ] update Figma
- [ ] update [Style Guide Presentation](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ)
- [ ] move the issue to the questions and review column and add the label `ready for product` 
Product
- [ ] product will update the staging deck (see slides above)

**Comment by georgeivasquez** [comment]
@ExperimentsInHonesty 

Figma file updated as needed.

**Comment by angelaliao1231** [comment]
Please provide update @georgeivasquez 

<details><summary>Instructions</summary>

1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."
</details> 

You can use this template
```
1. Progress: 
2. Blockers: 
3. Availability:
4. ETA:
5. Pictures (if necessary): 
```

**Comment by krystalphn** [comment]
@ExperimentsInHonesty 

Updated the Presentation Style Guide:
- [Drop shadow slides](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g2a5d13e8d2c_0_24#slide=id.g2a5d13e8d2c_0_24) (3 total)

**Comment by Rabia2219** [comment]
@krystalphn 
Please add your notes from your meeting with Bonnie yesterday...

**Comment by krystalphn** [comment]
After meeting summary:

Some details on the [Presentation Style Guide](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g2a5d13e8d2c_0_24#slide=id.g2a5d13e8d2c_0_24) are missing. I am in the process of updating the Style Guide, waiting on some clarifying answers from @georgeivasquez

**Comment by krystalphn** [comment]
Done thank you for reminding me
    On Wednesday, September 3, 2025 at 11:52:17 AM PDT, Rabia Shaikh ***@***.***> wrote:  
 
 Rabia2219 left a comment (hackforla/tdm-calculator#1998)
@krystalphn
Please add your notes from your meeting with Bonnie yesterday...

**Comment by Rabia2219** [comment]
@georgeivasquez Please provide update

<details><summary>Instructions</summary>

1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."
</details> 

You can use this template
```
1. Progress: 
2. Blockers: 
3. Availability:
4. ETA:
5. Pictures (if necessary): 
```

**Comment by krystalphn** [comment]
Update on this issue:

After revision, we opened a new issue #2646 waiting to close this one to then close issue #1998.

Also, with issue #1998 two slides were added to the Style Guide Presentation:
- Intro slide for  [Infoboxes](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g37db27bcff4_0_0#slide=id.g37db27bcff4_0_0)
- Intro slide for [Tooltips](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g2d6c1a3bc0a_0_0#slide=id.g2d6c1a3bc0a_0_0)

Could someone help me add these two slides on the Staging [deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g35c7465f46c_2_24#slide=id.g35c7465f46c_2_24) I'm not sure on where and how to put them.

**Comment by ExperimentsInHonesty** [comment]
@georgeivasquez This issue has two pieces of outstanding work to be done

First piece we moved to its own isssue: https://github.com/hackforla/tdm-calculator/issues/2646.  Its off your plate now.

The remaining item, is for you to complete.  When you are done, please put this into the questions column with a ready for product label.

- [ ] Make this change in the [style guide presentation, color palette](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g2e134a56948_3_13#slide=id.g2e134a56948_3_13)  and the [design system in Figma, color palette](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=24025-15669&t=NVF3rU0jd15s7bn9-0)

<img width="709" height="307" alt="Image" src="https://github.com/user-attachments/assets/72b6f12a-822c-4cc6-956b-df6b741417e9" />

**Comment by georgeivasquez** [comment]
@ExperimentsInHonesty 

Buttons have been updated and centered in both the [Design System file](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=3337-11981&p=f&t=qntcs4GUbUZpppcS-0) and the [Style Guide Stage deck](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g37b706653a8_1_1#slide=id.g37b706653a8_1_1).

Quick question — the right-aligned buttons from [Issue #1889](https://github.com/hackforla/tdm-calculator/issues/1889):
Do those final designs also need to be updated to center alignment in order to fully close out the issue?

**Comment by debangeedas** [comment]
#### Slides
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
   - ~[Staging Deck, slide 1998](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?pli=1#slide=id.g328276ff956_0_0)~
      - [staging deck, issue 1998, slide 1](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g35c7465f46c_2_24#slide=id.g35c7465f46c_2_24) - consistent with [Figma ](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=24025-14586&t=4r86eCHSJqZ8QOmB-4), minor change made to[ style guide slide](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g2e77662f2ce_0_10#slide=id.g2e77662f2ce_0_10) to make 100% consistent
         - a design review is needed, please add to design agenda https://github.com/hackforla/tdm-calculator/issues/2227#issuecomment-4263224889
      - [staging deck, issue 1998, slide 2](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g35d66aff36a_4_15#slide=id.g35d66aff36a_4_15) - consistent with [Figma ](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=24025-15669&t=4r86eCHSJqZ8QOmB-4)and [Style Guide slide](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g2e134a56948_3_13#slide=id.g2e134a56948_3_13)
      - [staging deck, issue 1998, slide 3](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g35d66aff36a_6_0#slide=id.g35d66aff36a_6_0) - consistent with [Figma ](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=24025-15669&t=4r86eCHSJqZ8QOmB-4)and [Style Guide slide](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g2e134a56948_3_13#slide=id.g2e134a56948_3_13)
      - [staging deck, issue 1998, slide 4](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g35d66aff36a_4_29#slide=id.g35d66aff36a_4_29) - consistent with [Figma](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=24025-15587&t=4r86eCHSJqZ8QOmB-4), not with Style guide (needs further inspection)
         - [x] check to see if possible update figma/styleguide/staging
         - [x] created issue #3127
      - [staging deck, issue 1998, slide 5](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g361502f6416_1_0#slide=id.g361502f6416_1_0) - consistent with [Figma](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=24025-14872&t=4r86eCHSJqZ8QOmB-4), not with [Style Guide slide](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g37b706653a8_1_1#slide=id.g37b706653a8_1_1)
         - [ ] check to see if possible update figma
            - [ ] When the figma check is done - Create a new issue to do the things indicated by the comments and text blocks on the slides of staging and styleguide.  This issue might also need changes to figma, but it has not been checked yet
         - [x] check to see if possible update styleguide
         - [x] check to see if possible update staging
      - [staging deck, issue 1998, slide 6](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g35eb1de5978_0_0#slide=id.g35eb1de5978_0_0) - consistent with [Figma](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=24025-14872&t=4r86eCHSJqZ8QOmB-4),  and with [Style Guide slide](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g37b706653a8_1_1#slide=id.g37b706653a8_1_1)
      - [staging deck, issue 1998, slide 7](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g35c7465f46c_2_50#slide=id.g35c7465f46c_2_50) - consistent with Style Guide slide, except this [slide](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g38931d8e49d_0_0#slide=id.g38931d8e49d_0_0) seems extra, similar in [Figma](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=24025-14872&t=4r86eCHSJqZ8QOmB-4)
      - [staging deck, issue 1998, slide 8](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g361502f6416_1_9#slide=id.g361502f6416_1_9) - consistent with [Style Guide slides](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g347b0193470_3_0#slide=id.g347b0193470_3_0), seems to be waiting for Bonnie's review. Figma does not have a dedicated section for drop shadows, but drop shadows seem to be integrated in the components themselves.
      - [staging deck, issue 1998, slide 9](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g361502f6416_1_35#slide=id.g361502f6416_1_35) - Consistent with [Style Guide slide](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g2f74a24ec8f_0_8#slide=id.g2f74a24ec8f_0_8) and [Figma](https://docs.google.com/presentation/d/1I4q35NL2WW2RpksIyawhHCd8qJnrjme1ZDNIBxu24hQ/edit?slide=id.g2f74a24ec8f_0_8#slide=id.g2f74a24ec8f_0_8)

**Comment by debangeedas** [comment]
- Opened new issue #3127

**Comment by ExperimentsInHonesty** [comment]
@debangeedas I reviewed slide 5 and added a checkbox for next steps on that (in your comment).  Ashish is going to finish the Figma review for5 and review the rest of the problems you identified.  When he is done with each slide, he will leave a comment for you to make the issues

**Comment by debangeedas** [comment]
- Opened new issue #3139
